script.js

```diff
@@ -435,14 +442,98 @@ const productsTable = {
   url: "./data/products.js"
 };
 
+const adminForm = {
+  view: "form",
+  id: "addCategoryForm",
+  gravity: 1,
+  minWidth: 180,
+  maxWidth: 250,
+  elements: [
+    { template: "EDIT CATEGORY", type: "section" },
+    {
+      view: "text",
+      label: "Category",
+      name: "value",
+      invalidMessage: "Must be filled"
+    },
+    {
+      margin: 20,
+      cols: [
+        {
+          view: "button",
+          label: "Add new",
+          css: "webix_primary",
+          click: function() {
+            this.getFormView().save();
+          }
+        },
+        {
+          view: "button",
+          label: "Clear",
+          click: () => {
+            confirmOfClearForm();
+          }
+        }
+      ]
+    },
+    {}
+  ],
+  rules: {
+    value: webix.rules.isNotEmpty
+  }
+};
+
+const adminDataTable = {
+  view: "datatable",
+  id: "categoryDataTable",
+  minWidth: 300,
+  fillspace: true,
+  select: true,
+  hover: "row--hover",
+  columns: [
+    { id: "id", header: "", sort: "int", width: 40, css: "column--id" },
+    {
+      id: "value",
+      header: [{ text: "Film category" }],
+      sort: "string",
+      adjust: true,
+      fillspace: true
+    },
+    {
+      header: "",
+      template: "<span class='remove-btn webix_icon wxi-trash'></span>"
+    }
+  ],
+  scrollX: false,
+  onClick: {
+    "remove-btn": function(e, id) {
+      filmCategoriesCollection.remove(id);
+      return false;
+    }
+  },
+  on: {
+    onAfterAdd: function(id) {
+      this.showItem(id);
+    }
+  },
+  ready: function() {
+    debugger;
+  }
+};
+
 const dashboard = {
   id: "dashboard",
   cols: [{ rows: [filterOptions, dataTable] }, form]
 };
 
+const admin = {
+  id: "admin",
+  cols: [adminDataTable, adminForm]
+};
+
 const multiView = {
   view: "multiview",
-  cells: [dashboard, users, productsTable]
+  cells: [admin, dashboard, users, productsTable]
```

Comment:
according to task, it is the last tab. 
moreover, you need to preselect the first menu item initially to indicate which multiview tab is open